### PR TITLE
Issue 7

### PR DESCRIPTION
**Pull request by DmitrySHilovsky**

### CODE REVIEW

File: tests/registration/registration.spec.ts [hunk 1]
```diff
@@ -16,14 +17,17 @@ test.describe(`Testing the entry positive and negative scenario`, async () => {
     await registrationPage.visitPage();
   });
 
-  test("Create new User", async () => {
+  test.only("Create new User", async () => {
```
Comment (vladimirchizhenkov) [inline tests/registration/registration.spec.ts]: Need to delete **only** method

File: tests/registration/registration.spec.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import test, { expect, Page } from "@playwright/test";
 import { RegistrationPage } from "../../src/components/pages/registration-page";
+import { timeout } from "../../src/support/utils";
 
 test.use({ viewport: { width: 1920, height: 1080 } });
```
Comment (vladimirchizhenkov) [inline tests/registration/registration.spec.ts]: Need to delete unused **timeout**

File: tests/registration/registration.spec.ts [hunk 3]
```diff
@@ -19,11 +20,14 @@ test.describe(`Testing the entry positive and negative scenario`, async () => {
...
+    expect(await registrationPage.getSuccessMessage()).
+    toBe('Your account was created successfully. You are now logged in.');
+   
+    expect(await registrationPage.getUserLogin()).
+    toBe(await registrationPage.getLoginFromWelcomeMessage());
   });
 
-  // ЗАДАЧА Оставить поля пустыми и кликнуть на кнопку регистрация, убедится что выдало ошибки и пользователь не зарегистрирован
   test("Create new User for empty Data", async () => {
     await registrationPage.clickRegistrationButton();
```
Comment (vladimirchizhenkov) [inline tests/registration/registration.spec.ts:30]: Need to delete empty end-strings 36, 37

File: tests/registration/registration.spec.ts (hunk 1 above)
Comment (DmitrySHilovsky) [reply tests/registration/registration.spec.ts]: fixed

File: tests/registration/registration.spec.ts (hunk 2 above)
Comment (DmitrySHilovsky) [reply tests/registration/registration.spec.ts]: fixed

**Review by vladimirchizhenkov — APPROVED** [review]

**Pull request merged**